src/h3/util.pyx

```diff
@@ -25,7 +25,7 @@ def versions():
     return v
 
 
-cpdef H3int hex2int(H3str h):
+cpdef H3int hex2int(H3str h) except *:
```

Comment:
Per https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html could `except? 0` be used here?

Reply:
good point! I'll change that

Reply:
Might also apply in other places like `src/h3/util.pyx ` line 75, but that's not new in this PR.

Reply:
Oooh, I totally didn't realize we had the `except?` syntax!

I'm now using `cpdef H3int hex2int(H3str h) except 0`, which should work, assuming that no valid H3 index can ever be zero.

In `util.py`, I'm thinking of using `cdef H3int* create_ptr(size_t n) except? NULL`, since `NULL` could potentially be a valid output if `n = 0`.

For the two examples...
- `edges.pyx`, `cpdef (H3int, H3int) edge_cells(H3int e) except *:`
- `geo.pyx`, `cpdef (double, double) h3_to_geo(H3int h) except *:`

...I think we have to stick with the `except *`, as I can't figure out how to get the syntax to work with anything else for multiple return values.

I would think that multiple return values would evaluate to a tuple (which is a Python object) making the `except *` unnecessary, but that doesn't see to be the case (the tests expecting the error to appear fail). And something like `except (0,0)` doesn't seem to work either. Happy to change it if anyone knows a better way!